superbit_lensing/shear_profiles/shear_plots.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+from matplotlib import rc,rcParams
+import matplotlib.pyplot as plt
+from astropy.table import Table
+import pdb
+
+class ShearProfilePlotter(object):
+
+    def __init__(self, cat_file, truth_file, pix_scale=0.144):
+        '''
+        cat_file: str
+            Filename for...
+        truth_file: str
+            Filename for nfw truth shear profile
+        pix_scale: float
+            Pixel scale (arcsec / pix)
+        '''
+
+        self.cat_file = cat_file
+        self.truth_file = truth_file
+        self.pix_scale = pix_scale
+
+        self._load_cats()
+
+        return
+
+    def _load_cats(self):
+        self.cat = Table.read(self.cat_file)
+        self.truth = Table.read(self.truth_file)
+
+        return
+
+    def get_angular_radius(pix_radius, arcmin=True):
+        angular_radius = pix_radius * self.pix_scale
+
+        if arcmin is True:
+            return angular_radius * 60.
+        else:
+            # in arcsec
+            return angular_radius
+
+    def plot_tan_profile(self, title=None, size=(10,7),
+                         label='annular', show=False, outfile=None):
+        rc('font', **{'family':'serif'})
+        rc('text', usetex=True)
+        plt.ion()
+
+
+        radius = get_angular_radius(self.cat['r'], arcmin=True)
+        gtan = self.cat['gtan']
+        gcross = self.cat['gcross']
+        gtan_err = self.cat['gtan_err']
+        gcross_err = self.cat['gcross_err']
+
+        try:
+            data.sort('col1') # get in descending order
+            ## uncomment below if it's needed
+            #data.remove_rows([0])
+            radius=data['col1']
+            radius=radius*pixscale/60.
+        except ValueError:
+            data.sort('r') # get in descending order
+            ## uncomment below if it's needed
+            #data.remove_rows([0])
+            radius=data['r']
+            radius=radius*pixscale/60.
+
+        try:
+            etan=data['gtan']
+            ecross=data['gcross']
+            shear1err=data['err_gtan']
+            shear2err=data['err_gcross']
+
+        except KeyError:
```

Comment:
This try/except block was useful when I was still using `.fiat` catalogs and hadn't figured out how to make `astropy.Table` interpret the column names. I'm not sure it's necessary now. For one thing, there's no guarantee that `col1`...`col6` of the input table actually correspond to `etan`...`shear2err`. Maybe we can remove them?

Reply:
I will add this to the "enhancements" list